lib/ui/anytime_podcast_app.dart

```diff
@@ -247,11 +247,14 @@ class _AnytimeHomePageState extends State<AnytimeHomePage> {
     final pager = Provider.of<PagerBloc>(context);
     var settings = Provider.of<SettingsBloc>(context).currentSettings;
     final backgroundColour = Theme.of(context).scaffoldBackgroundColor;
+    final isLight = Theme.of(context).brightness == Brightness.light;
 
     return AnnotatedRegion<SystemUiOverlayStyle>(
       value: SystemUiOverlayStyle(
-        statusBarIconBrightness: Theme.of(context).brightness == Brightness.light ? Brightness.dark : Brightness.light,
-        systemNavigationBarColor: Theme.of(context).bottomAppBarColor,
+        statusBarIconBrightness: isLight ? Brightness.dark : Brightness.light,
+        systemNavigationBarColor: isLight
```

Comment:
does this change also works (fixes) anytime without breez?

Reply:
Yes because we want to push this PR to anytime as well.

Reply:
Just test and it is working :)
I'll open this PR on anytime repo as well